src/doc/using_visit/SavingPrinting/Saving_movies.rst

```diff
@@ -190,11 +139,251 @@ click the **Movie progress dialog's** **Cancel** button.
+
+Slice 3D Movie Template
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The **Slice 3D** movie template animates a slice plane through all three dimensions of a 3D dataset before fading to black.
+
+.. _slice3d_preview:
+
+.. figure:: images/slice3d_preview.png 
+   :scale: 75%
+
+   Slice 3D template preview.
+
+
+The next page (:numref:`Figure %s<slice3d_options>`) lets you fill in details such as which dataset will be sliced.
+You can also set visualization title and colors.
+
+.. _slice3d_options:
+
+.. figure:: images/slice3d_options.png 
+   :scale: 75%
+
+   Slice 3D parameters.
+
+This movie template ends with a fade to black sequence.
+You can set the duration of this sequence as well as the colors used as shown in :numref:`Figure %s<slice3d_fade_options>`.
+   
+.. _slice3d_fade_options:
+
+.. figure:: images/slice3d_fade_options.png 
+   :scale: 75%
+
+   Slice 3D fade sequence options.
+
+
+Creating new Templates
+""""""""""""""""""""""
+
+.. _savemovie_createtemplate:
+
+.. figure:: images/savemovie_createtemplate.png 
+   :scale: 75%
+   
+   Creating a new template
+
+.. _SidebySideMovie:
+
+Side by Side Movie
+^^^^^^^^^^^^^^^^^^
+To make a side by side movie you can use the **Create template** option.
+Before saving the movie with this template, you must set up the plots that you want to make in multiple vis windows, as shown in :numref:`Figure %s<sidebyside_plots>`.
+
+.. _sidebyside_plots:
+
+.. figure:: images/sidebyside_plots.png 
+   :scale: 75%
+   
+   Setting up plots for Side by Side movie generation.
+
+Since you are creating a new movie template, you first must set up the movie viewports.
+You can create many viewports and the viewports can be adjacent or can overlap (e.g. picture in picture).
+When viewports overlap, you can make the top viewports semi-transparent, make a background color transparent, or even add drop shadows.
+There are some existing layouts if you don't want to get fancy.
+Select the **Side by Side** layout and VisIt will create 2 viewports next to one another.
+
+.. _sidebyside_layout_selection:
+
+.. figure:: images/sidebyside_layout_selection.png 
+   :scale: 75%
+   
+   Selecting Side by Side layout.
+
+.. _sidebyside_viewports:
+
+.. figure:: images/sidebyside_viewports.png 
+   :scale: 75%
+   
+   The Side by Side viewports.
+
+After you create movie viewports, you must map sequences of images to the viewports.
+Sequences are series of images and they can be produced from the plots in your vis windows or transitions such as fades, etc.
+For a side by side movie, you will want to map the image sequence produced from vis window 1 to movie viewport 1.
+You will also map the image sequence from vis window 2 to movie viewport 2.
+If you had more vis windows, you could map more than one vis window to a movie viewport.
+That has the effect that all images from a particular vis window are used as inputs for the movie viewport and then the next vis window that was mapped to the movie viewport, and so on.
```

Comment:
This sentence confused me. Do you mean that ensuing vis windows will be used as inputs for the movie viewport as well?

Reply:
Confusing to me as well, I grabbed it from Side by Side movie page on visitusers. Should I just remove that sentence?

Reply:
I'm reading it again and the more I read it, the less I understand what it is trying to say. If you have some idea of what it is supposed to say then I would advocate for replacing it, but as it stands it doesn't seem to be adding anything. Removing it seems like a decent idea.

Reply:
How about (I just tried this to see what would happen):

When more than one vis window is mapped to the same viewport, the movie will contain the animation from the first vis window followed by the animation from the second vis window.

Reply:
That sounds good.